app/graphql/extensions/authenticable.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Extensions
+  module Authenticable
+    def user_sign_in?
+      current_user.present?
+    end
+
+    def authenticate_user
+      raise GraphQL::ExecutionError, I18n.t('auth.errors.authenticate') unless user_sign_in?
```

Comment:
let`s better use root path at i18n. I mean: `I18n.t('.auth.errors.authenticate')` - with dot on the start of path